tests/test_0031-test-stl-containers.py

```diff
@@ -14,6 +14,31 @@
 from uproot.interpretation.objects import AsObjects
 
 
+@pytest.mark.skip(
+    reason="Implement non-memberwise std::map; we have a sample (map<string,double>)"
+)
+def test_nonmemberwise_asmap():
```

Comment:
This is for me to implement non-memberwise `std::map` in Python, so that you can develop Forth code for both the memberwise and non-memberwise cases. (Just for `std::map`.) These four tests will be added to test_0637.